### ISSUE TITLE
Add pouchdb-find support

### ISSUE THREAD

**Issue by nolanlawson** [nolanlawson/worker-pouch#2]
Should be easy by just adding some kind of custom implementation capability like I did for map/reduce.

**Comment by ronyrun** [comment]
I'm able to use the db.find with worker.

**Comment by nolanlawson** [comment]
Yeah, but I believe if could be more efficient if worker-pouch included find support. Right now it still has to execute a lot of queries between the UI thread and the worker thread, when it could isolate more of them to the worker thread.

**Comment by ronyrun** [comment]
:+1:

**Comment by jorfermo** [comment]
+1

**Comment by NNemec** [comment]
Does not work for me at all. Pity - this makes this promising project unusable to me. :-(

**Comment by kernelwhisperer** [comment]
> Yeah, but I believe if could be more efficient if worker-pouch included find support. Right now it still has to execute a lot of queries between the UI thread and the worker thread, when it could isolate more of them to the worker thread.

I would like to tackle this issue and make a PR. 
Could you elaborate a bit more and give me some pointers? Thanks! 🙏
